src/libsync/theme.cpp

```diff
@@ -412,5 +412,10 @@ QString Theme::webDavPath() const
     return QLatin1String("remote.php/webdav/");
 }
 
+QString Theme::webDavPathNonShib() const
+{
+    return QLatin1String("remote.php/nonshib-webdav/");
```

Comment:
This is the default alternative path for webdav clients that do not implement Shibboleth authentication in case the server has Shibboleth Integration, hence the "nonshib". See https://doc.owncloud.org/server/9.0/admin_manual/enterprise_user_management/user_auth_shibboleth.html#webdav-support